content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -44,6 +44,14 @@ via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the
     $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml -n istio-system
     {{< /text >}}
 
+__NOTE__: If you are enabling `certmanager`, you also need to install its CRDs as well and wait a few seconds for the CRDs to be committed in the kube-apiserver:
+
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/istio/charts/certmanager/templates/crds.yaml
+    {{< /text >}}
+
+This limitation should be removed when Istio upgrade Helm to version 2.10 and above.
```

Comment:
please don't include forward looking content in the documentation.  We can easily backport a change to the documentation when helm 2.10 is available (which is shortly from my understanding)